src/classes/TDTM_DefaultConfig.cls

```diff
@@ -333,6 +333,11 @@ public without sharing class TDTM_DefaultConfig {
                 Class__c = 'CRLP_Rollup_TDTM', Load_Order__c = 2, Object__c = 'npe01__OppPayment__c',
                 Trigger_Action__c = 'AfterInsert;AfterUpdate;AfterDelete;AfterUndelete'));
 
+        // Customizable Rollups on Account Soft Credit
```

Comment:
I was going to say that the trigger needs to be removed, however that's not 100% correct. The trigger should *not* attempt to roll up Account Soft Credit data as previously noted, but what we do want to do is have logic to ensure that the CurrencyIsoCode on the AccountSoftCredit object is inherited from the parent Opportunity object. You can find some similar logic on the Payment, Allocation and hopefully PartialSoftCredit objects.

Reply:
It's relatively trivial to add this to the existing PSC_Opportunity_TDTM class. Is having it buried in there to share code acceptable, or should this have its own namespace and class?